packages/core/src/odata-common/request-builder/request-builder-base.ts

```diff
@@ -132,6 +132,16 @@ export abstract class MethodRequestBuilder<
     return this;
   }
 
+  /**
+   * Replace the default value of fetching csrf token configuration with a given value.
```

Comment:
Perhaps we change the API to `disableCsrfTokenFechting()` without the argument. Most likely a `enavleCsrfTokenFetching` method is not needed because why would you disable it in the frist place. But I am also fine with this API, just mention the default value in the docs.

Reply:
@marikaner what is your opinion on this?

Reply:
Yes, you are right. For vdm request builder level, the default behaviour should always be "enable the csrf token fetching". Therefore, having one API called disable*** is fine. I'll adjust it.

Reply:
+1